### ISSUE TITLE
Accessibility progress not calculated when re-editing accessibility settings

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#27]
## Expected Behavior

Validation status should be calculated when accessibility form is opened to edit previous configuration

## Current Behavior

Validation status is not calculated

![image](https://user-images.githubusercontent.com/993369/76171050-2bf11600-6166-11ea-9fdf-5ea8d02d3ffe.png)

## Steps to Reproduce

1. Configure accessibility for image
2. Save
3. Select to configure accessibility for the same image
4. Form is opened with wrong accessibility status

## Possible Implementation

Instead of load isA11y variable from database, try to call the function to calculate it.
Also, check if isA11y is being persisted correctly.

**Comment by CAndresH** [comment]
This is solved now
